README.md

```diff
@@ -1,64 +1,62 @@
+build Docker images for Java applications.
 
-Please be aware that:
-
-* we are providing jre images based on different OS but at the time of writing
-  only `jre11-centos7` is passing security scan on our private quay.io. You can
-  check for updated security scans results on the quay.io [image
-  page](https://quay.io/repository/alfresco/alfresco-base-java?tab=tags).
-
-## Quickstart
+## Flavours
 
 Choose between one of the available flavours built from this repository:
 
-Java version | Java flavour | OS            | Image tag         | Size
--------------|--------------|---------------|-------------------|-----------------------------
-11           | jre          | Centos 7      | jre11-centos7     | ![jre11-centos7 size][1]
-17           | jre          | Rocky Linux 8 | jre17-rockylinux8 | ![jre17-rockylinux8 size][2]
-11           | jre          | Rocky Linux 8 | jre11-rockylinux8 | ![jre11-rockylinux8 size][3]
-17           | jre          | Rocky Linux 9 | jre17-rockylinux9 | ![jre17-rockylinux9 size][4]
+Java version | Java flavour | OS            | Image ref                                       | Size
+-------------|--------------|---------------|-------------------------------------------------|-----------------------------
+11           | jre          | Centos 7      | `alfresco/alfresco-base-java:jre11-centos7`     | ![jre11-centos7 size][1]
+17           | jre          | Rocky Linux 8 | `alfresco/alfresco-base-java:jre17-rockylinux8` | ![jre17-rockylinux8 size][2]
+11           | jre          | Rocky Linux 8 | `alfresco/alfresco-base-java:jre11-rockylinux8` | ![jre11-rockylinux8 size][3]
+17           | jre          | Rocky Linux 9 | `alfresco/alfresco-base-java:jre17-rockylinux9` | ![jre17-rockylinux9 size][4]
 
 [1]: https://img.shields.io/docker/image-size/alfresco/alfresco-base-java/jre11-centos7
 [2]: https://img.shields.io/docker/image-size/alfresco/alfresco-base-java/jre17-rockylinux8
 [3]: https://img.shields.io/docker/image-size/alfresco/alfresco-base-java/jre11-rockylinux8
 [4]: https://img.shields.io/docker/image-size/alfresco/alfresco-base-java/jre17-rockylinux9
 
-* [Docker Hub](https://hub.docker.com/r/alfresco/alfresco-base-java) image name: `alfresco/alfresco-base-java`
-* [Quay](https://quay.io/repository/alfresco/alfresco-base-java) image name: `quay.io/alfresco/alfresco-base-java`
+The images are available on:
 
-Example final image: `alfresco/alfresco-base-java:jre11-centos7`
+* [Docker Hub](https://hub.docker.com/r/alfresco/alfresco-base-java), image name: `alfresco/alfresco-base-java`
+* [Quay](https://quay.io/repository/alfresco/alfresco-base-java) (enterprise credentials required), image name: `quay.io/alfresco/alfresco-base-java`
 
-> If you are using this base image in a public repository, please stick to the DockerHub published image.
+> If you are using this base image in a public repository, please use the Docker
+> Hub hosted one
```

Comment:
Not sure what we meant here...

Reply:
to not use quay on public/opensource docker projects, but I guess it's already clear since I added the credentials requirements bit